vms/platformvm/genesis/camino.go

```diff
@@ -102,47 +106,55 @@ func (offer DepositOffer) Verify() error {
 		)
 	}
 
+	if offer.OfferID != ids.Empty {
```

Comment:
In which case offerID will be empty? Can't we always expect right offerID specified in json, so it will eventually get here?

Reply:
The templates JSON won;t have offerIDs because they don't have an "final" value (which depends on startTime)